drms/tests/test_to_datetime.py

```diff
@@ -52,6 +58,20 @@ def test_force_string(time_string, expected):
     assert drms.to_datetime(time_string, force=True) == expected
 
 
-# test_xx_list
-# test_xx_ndarray
-# test_xx_pandas_series
+@pytest.mark.parametrize('time_series, expected', [
+    (data_tai_in, data_tai_out),
+    (pd.Series(data_tai_in), data_tai_out),
+    (tuple(data_tai_in), data_tai_out),
+    (np.array(data_tai_in), np.array(data_tai_out)),
+    (dict(zip(tuple(range(len(data_tai_in))), tuple(data_tai_in))),
+     dict(zip(tuple(range(len(data_tai_out))), tuple(data_tai_out))))
+    ])
+def test_time_series(time_series, expected):
+    drms.to_datetime(time_series).equals(expected)
+
+
+def test_corner_case():
+    assert type(drms.to_datetime('')) == pd._libs.tslib.NaTType
+    assert type(drms.to_datetime('foo')) == pd._libs.tslib.NaTType
+    assert type(drms.to_datetime([])) == pd.Series
+    assert drms.to_datetime([]).empty
```

Comment:
You can check if an object is an instance of a certain class by using the built-in `isinstance()` function. So instead of
```python
assert type(drms.to_datetime([])) == pd.Series
```
you should write
```python
assert isinstance(drms.to_datetime([]), pd.Series)
```

`pandas.NaT` values behave similar as `NaN` values. For example the expression `NaT != NaT` is always true. To check for `NaN` in NumPy, one usually uses `numpy.isnan()`. I think there does not exist a function in Pandas that can explicitly check for `NaT` (like `pandas.isnat()` or similar), but you can always use `pandas.isnull()` to check for invalid/missing values. This function would also return `True` for `NaN` or `None` values, but this should not be a problem in this case.

Alternatively you could also use `isinstance(x, type(pandas.NaT))` or `isinstance(x, pandas.NaTType)`, but I would just use `pandas.isnull()` instead.